### ISSUE TITLE
Error with react 18

### ISSUE THREAD

**Issue by plantain-00** [pixijs/pixi-react#337]
### Description

seems related: https://github.com/inlet/react-pixi/issues/306

```
react-dom.development.js:22665 Uncaught Error: Expected subtree parent to be a mounted class component. This error is likely caused by a bug in React. Please file an issue.
    at findCurrentUnmaskedContext (react-pixi.es-dev.js:3904:15)
    at getContextForSubtree (react-pixi.es-dev.js:20484:23)
    at Object.updateContainer (react-pixi.es-dev.js:20603:17)
    at Stage.componentDidMount (react-pixi.es-dev.js:22880:17)
```

### Steps to reproduce

1. Use the code from https://reactpixi.org/components/graphics
2. The error above appears.

### Additional info

- `@inlet/react-pixi` version: 6.7.0
- `React` version: 18.0.0
- `ReactDOM` version: 18.0.0
- `PIXI` version: 6.3.0

**Comment by jdlee022** [comment]
Is there a solution for this?

**Comment by plantain-00** [comment]
> Is there a solution for this?

I couldn't find any solution in the past several months, so I migrated to raw WebGL instead.

**Comment by saitonakamura** [comment]
@plantain-00 @jdlee022  You can use `@saitonakamura/react-pixi@17.0.0-beta-5`, it has support for react 18 and we've being using it in production for a month

**Comment by baseten** [comment]
This should be fixed now in `v7.0.0-alpha.0`, please install from `@pixi/react-pixi`
